distributions/Jamulus.1

```diff
@@ -227,48 +227,47 @@ left to right.
 does not provide feedback as to the current state of the Solo and Mute
 buttons so the controller must track and signal their state locally.
 .It Fl \-directoryfile Ar file
-.Pq directory server mode only
-enable server list persistence, storing to and loading from
-.Ar file
+.Pq Directory Server mode only
+remember registered Servers even if the Directory is restarted
 .It Fl \-mutemyown
-.Pq headless client only
+.Pq headless Client only
 mute my channel in my personal mix
 .It Fl \-norecord
-.Pq server mode only
+.Pq Server mode only
 do not automatically start recording even if configured with
 .Fl R
 .It Fl \-serverbindip Ar ip
-.Pq server mode only
+.Pq Server mode only
 configure Legacy IP address to bind to
 .It Fl \-serverpublicip Ar ip
-.Pq server mode only
-configure public Legacy IP address when both the directory server
-and the actual server are situated behind the same NAT, so that
-clients can connect
+.Pq Server mode only
+configure public legacy IP address when both the Directory Server
```

Comment:
Why is the word "legacy" here at all?

Reply:
“Legacy IP” is the contemporary wording for IPv4 (IP is IPv6). This was introduced into networking terminology a small number of years ago.

Reply:
So that all us old timers _feel_ old...